dali/operators/reader/loader/video/frames_decoder_gpu.cc

```diff
@@ -137,7 +137,7 @@ class NVDECCache {
             it->second.bit_depth_luma_minus8 == bit_depth_luma_minus8) {
           it->second.used = true;
           assert(it->second.device_id == device_id);
-          return NVDECLease(it->second);
+          return NVDECLease(this, &it->second);
```

Comment:
There is unchanged return in line 160. It is under the `ifdef ENABLE_NVDEC_RECONFIGURE`, so it might have been not verified.

Reply:
This part need to be revisited before reenabling. It is more like a note than the actual working code now.

Reply:
Nice catch! Fixed.